Neos.Media/Migrations/Mysql/Version20191125132700.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Flow\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Migrations\AbortMigrationException;
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Set default for ratio mode for image adjustments
+ */
+class Version20191125132700 extends AbstractMigration
+{
+
+    /**
+     * @return string
+     */
+    public function getDescription(): string
+    {
+        return 'Set default for ratio mode for image adjustments';
+    }
+
+    /**
+     * @param Schema $schema
+     * @return void
+     * @throws DBALException
+     * @throws AbortMigrationException
+     */
+    public function up(Schema $schema): void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on "mysql".');
+        $this->addSql('UPDATE neos_media_domain_model_adjustment_abstractimageadjustment SET ratiomode=\'inset\' WHERE ratiomode IS NULL AND dtype=\'neos_media_adjustment_resizeimageadjustment\'');
```

Comment:
Hi, I have followed your bugfix PR @kdambekalns. 
Thx for your fast reaction. 
I have seen that you use dtype=\'neos_media_adjustment_resizeimageadjustment\' but what is with e.g. neos_media_adjustment_cropimageadjustment.